src/App.jsx

```diff
@@ -14,45 +16,96 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button
-        type="button"
-        className="button is-info is-light"
-      >
-        Sort alphabetically
-      </button>
-
-      <button
-        type="button"
-        className="button is-success is-light"
-      >
-        Sort by length
-      </button>
-
-      <button
-        type="button"
-        className="button is-warning is-light"
-      >
-        Reverse
-      </button>
-
-      <button
-        type="button"
-        className="button is-danger is-light"
-      >
-        Reset
-      </button>
-    </div>
+const buttons = [
+  {
+    name: 'Sort alphabetically',
+    color: 'is-info',
+  },
+  {
+    name: 'Sort by length',
+    color: 'is-success',
+  },
+  {
+    name: 'Reverse',
+    color: 'is-warning',
+  },
+];
+
+export const App = () => {
+  const [stateSort, setStateSort] = useState(null);
+  const [stateReverse, setStateReverse] = useState(null);
+  let goods = [...goodsFromServer];
+
+  switch (stateSort) {
+    case 'Sort alphabetically':
+      goods = goods.sort((good1, good2) => good1.localeCompare(good2));
+      break;
+    case 'Sort by length':
+      goods = goods.sort((good1, good2) => good1.length - good2.length);
+      break;
+    case null:
+      goods = [...goodsFromServer];
+      break;
+    default:
+      goods = [...goodsFromServer];
+  }
+
+  if (stateReverse) {
+    goods = goods.reverse();
+  }
```

Comment:
More correctly will be wrap this part of code in function and move it outside the component